packages/test/local-server-tests/package.json

```diff
@@ -26,7 +26,7 @@
     "lint:fix": "npm run eslint:fix",
     "test": "npm run test:mocha",
     "test:coverage": "cross-env FLUID_TEST_COVERAGE=1 npm run test:mocha",
-    "test:mocha": "mocha dist/test",
+    "test:mocha": "mocha dist/test -r node_modules/@fluidframework/mocha-test-setup --unhandled-rejections=strict --exit",
```

Comment:
these are specified in the .mocharc.js

Reply:
https://github.com/curtisman/FluidFramework/blob/c2d9b597178a00a134e412314e7dc83d0b203ed0/packages/test/local-server-tests/.mocharc.js

Reply:
personally, i'd rather standardize on using a config file, as otherwise we end up using `--` a bunch to force down dynamic options, that we can more easily make dyanmic in the config